.travis.yml

```diff
@@ -107,7 +107,8 @@ jobs:
 stages:
   - name: test
     if: type = pull_request or (type = push and branch = master)
-  - name: deploy
-    if: type = push and tag =~ ^v
+  # Until deployment of the release binaries is fixed, always build them
```

Comment:
It don't understand this sentence, also seems unrelated to LTO ;).
Disabling auto-deploy to GH releases seems fine for now, as mentioned earlier I'd prefer to use the official binaries instead of rebuilding from sources. IIUC this PR adds support for LDC building and the GH binaries would prolly use it, while the official binaries still use dmd.
This inconsistency is likely to require additional maintenance and debug efforts down the road. Would be slightly cleaner to switch over dub together with using gdc or ldc to build optimized dmd release binaries.

In fact integrating gdc or ldc with [installer/create_dmd_release](https://github.com/dlang/installer/tree/6763102708fbf6ff59bd13ed6cf761aa05c0e07c/create_dmd_release) just for dub would be a good first step towards changing the optimized compiler for dmd releases.